src/vectorclock.erl

```diff
@@ -68,6 +114,51 @@ max([]) -> new();
 max([V]) -> V;
 max([V1,V2|T]) -> max([merge(fun erlang:max/2, V1, V2)|T]).
 
+-spec max_vc(vectorclock(), vectorclock()) -> vectorclock().
```

Comment:
I don't like the names `max_vc` and `min_vc`. It is easy to confuse this with the `max` operation, which computes the least upper bound of two vectorclocks. I would also expect that a function named `max_vc` is symmetric. 
Wouldn't it be enough to optimize the max function for merging two vectorclocks? And why would a vectorclock ever match with `ignore`, `undefined` or `[]`?